packages/mock-block-dock/src/mock-block-dock.tsx

```diff
@@ -46,131 +41,25 @@ export const MockBlockDock: VoidFunctionComponent<MockBlockDockProps> = ({
   initialLinks,
   initialLinkedAggregations,
 }) => {
-  const mockData = useMemo((): MockData => {
```

Comment:
this has all been extracted into a hook, so this component now only gets stuff from the hook and injects it into the child